xwrf/grid.py

```diff
@@ -0,0 +1,139 @@
+"""Provide grid-related functionality specific to WRF datasets.
+
+This submodule contains code reused with modification from Salem under the terms of the BSD
+3-Clause License. Salem is Copyright (c) 2014-2021, Fabien Maussion and Salem Development Team All
+rights reserved.
+"""
+
+import numpy as np
+import pyproj
+
+# Default CRS (lon/lat on WGS84, which is EPSG:4326)
+wgs84 = pyproj.CRS(4326)
+
+# WRF dimension names
+_HORIZONTAL_DIMS = ['south_north', 'west_east', 'south_north_stag', 'west_east_stag']
+
+
+def _wrf_grid_from_dataset(ds):
+    """Get the WRF projection and dimension coordinates out of the file."""
+
+    pargs = {'x_0': 0, 'y_0': 0, 'a': 6370000, 'b': 6370000}
+    if hasattr(ds, 'PROJ_ENVI_STRING'):
+        # HAR and other TU Berlin files
```

Comment:
I think you can safely remove this case which is very particular to a dataset we generated in Berlin